src/stac_utils/email.py

```diff
@@ -28,6 +31,17 @@ def send_email(
         template: str = None,
         variables: dict = None,
     ):
+        """
+        Sends email given specified details
+
+        :param subject: Desired subject for email
+        :param body: Body of email
+        :param emails: List of email addresses receiving email
+        :param from_addr: From address for email
+        :param reply_to: Reply-to for email
+        :param template: Specified template for email
+        :param variables: Specified variables for email template
+        """
```

Comment:
i might specify that you can either include the body with the raw html directly, or the template + variables so it'll use the template